### ISSUE TITLE
Error executing vim.schedule: attempt to index field 'callbacks'

### ISSUE THREAD

**Issue by s-rafaeldias** [nvim-lua/completion-nvim#269]
**My testing minimal init.vim**
``` 
call plug#begin('~/.vim/plugged')
    Plug 'neovim/nvim-lsp'
    Plug 'nvim-lua/completion-nvim'
call plug#end()
lua require'lspconfig'.pyls.setup{on_attach=require'completion'.on_attach}
au Filetype lua setl omnifunc=v:lua.vim.lsp.omnifunc
" Use <Tab> and <S-Tab> to navigate through popup menu
inoremap <expr> <Tab>   pumvisible() ? "\<C-n>" : "\<Tab>"
inoremap <expr> <S-Tab> pumvisible() ? "\<C-p>" : "\<S-Tab>"

" Set completeopt to have a better completion experience
set completeopt=menuone,noinsert,noselect
```

**How to reproduce**
1. Install https://github.com/palantir/python-language-server
2. `nvim -u test.vim`
3. open any python file
4. type anything that should trigger an autocomplete such as `from da`
5. Should get the error: `Error executing vim.schedule lua callback: ...el/.vim/plugged/completion-nvim/lua/completion/hover.lua:307: attempt to index field 'callbacks' (a nil value)`

**Expected behavior**
To no get any error.

**Comment by lithammer** [comment]
Fixed by #266, and caused by refactoring in https://github.com/neovim/neovim/pull/12655.

**Comment by lithammer** [comment]
Or actually, seems like it's already fixed by https://github.com/nvim-lua/completion-nvim/pull/268. Try updating completion-nvim?

**Comment by s-rafaeldias** [comment]
Just updated here and fix it!

**Comment by s-rafaeldias** [comment]
The fix on #268 solved this.
